tests/test-sessions/test-infinispan-sessions/src/test/java/org/eclipse/jetty/server/session/InfinispanSessionDataStoreTest.java

```diff
@@ -164,30 +166,26 @@ public boolean checkSessionPersisted(SessionData data) throws Exception
     @Test
     public void testQuery() throws Exception
     {
-        Cache<String, SessionData> cache = _testSupport.getCache();
+        InfinispanSessionData sd1 = new InfinispanSessionData("sd1", "", "", 0, 0, 0, 1000);
```

Comment:
Why did you change the test data so that all will have expired?

Reply:
The previous testQuery didn't work from the beginning, so I replaced it with the test used in RemoteInfinispanSessionDataStoreTest

Reply:
Fair enough.